src/runtime/manifest-ast-nodes.ts

```diff
@@ -394,11 +394,14 @@ export interface ParticleConnectionTargetComponents extends BaseNode {
 
 export type RecipeHandleFate = string;
 
+export type RecipeHandleCapability = string;
```

Comment:
Just so I don't forget, I think we be more explicit about the type here, listing the string value options, this should improve our type safety and save us from having to handle impossible cases in ts.

Reply:
added a list of values here. is that what you meant?